README.md

```diff
@@ -64,11 +64,16 @@ declaration.
 When the `-tiny` flag is passed, extra information is stripped from the resulting
 Go binary. This includes line numbers, filenames, and code in the runtime that
 prints panics, fatal errors, and trace/debug info. All in all this can make binaries
-2-5% smaller in our testing.
+2-5% smaller in our testing, as well as prevent extracting some more information.
 
-Note: if `-tiny` is passed, no panics or fatal errors will ever be printed, but they can
-still be handled internally with `recover` as normal. In addition, the `GODEBUG`
-environmental variable will be ignored.
+With this flag, no panics or fatal runtime errors will ever be printed, but they
+can still be handled internally with `recover` as normal. In addition, the
+`GODEBUG` environmental variable will be ignored.
+
+Note that this flag can make debugging crashes harder, as a panic will simply
+exit the entire program without printing a stack trace, and source position
```

Comment:
I don't think it's accurate to say source position info is entirely gone, because it's still there, just set to 1. Additionally, I have seen some tools recover some data regarding a function's length, though the line numbers always started at 1. 

Ie, would say that `func foo()` is 96 lines long, from lines 1-96. Tool I tested with is https://github.com/goretk/redress

Reply:
> I don't think it's accurate to say source position info is entirely gone, because it's still there, just set to 1.

Right, though I guess that's a technicality. Can a reverse-engineering tool tell you anything if all positions are 1, versus 0 or missing?

> Ie, would say that `func foo()` is 96 lines long, from lines 1-96. Tool I tested with is https://github.com/goretk/redress

How does it figure that out? That's probably something we should fix.

Reply:
That is a technicality, but because we did actually remove all position info before with `-tiny`, I think it's worth specifying that we don't remove the position info, we just set everything to 1.

I'm not sure how redress computes function length, but it's not very accurate. That's a separate issue anyway

Reply:
Fair enough, now it says:

> and all source code positions are set to line 1

--

> I'm not sure how redress computes function length, but it's not very accurate. That's a separate issue anyway

It might estimate that based on the Go statements it re-constructs.